packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -571,16 +571,27 @@ export function listOperationsInOperationGroup(
   return operations;
 }
 
+interface CreateSdkContextOptions {
+  ignoreVersioning?: boolean;
```

Comment:
```suggestion
  readonly ignoreVersioning?: boolean;
```

could consider making this a union of string values in case there is different strategies in the future.